src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps2_mqtt_client_module.c

```diff
@@ -121,7 +209,7 @@ void ADPS_MQTT_Client_Module_OnPublish(
 
 const char* topicArray[1] = { "$dps/registrations/res/#" };
 
-static bool ADPS_MQTT_Client_Module_GetSubscriptTopics(int* count, char*** topics)
+static bool ADPS_MQTT_Client_Module_GetSubscriptTopics(void* obj, int* count, char*** topics)
 {
     bool success = false;
     if (count == NULL || topics == NULL)
```

Comment:
obj arg not used--can it be removed from fn signature? Should it fail if NULL obj is passed too?